<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run | Programs </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Run Programs&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Scenarios&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_help">
         <property name="text">
          <string notr="true">Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTreeView" name="scenarios_tree"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>File Type</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="rd_allprograms">
                <property name="text">
                 <string>All Programs</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rd_selected_programs">
                <property name="text">
                 <string>Selected programs</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chck_path_search">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">margin-left: 20</string>
                </property>
                <property name="text">
                 <string>Path search</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chck_initial_assignment">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">margin-left: 20</string>
                </property>
                <property name="text">
                 <string>Initial Assignment</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chck_location">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">margin-left: 20</string>
                </property>
                <property name="text">
                 <string>Location</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chck_fixed_transportable">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">margin-left: 40</string>
                </property>
                <property name="text">
                 <string>Fixed transportables</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chck_assignment">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">margin-left: 20</string>
                </property>
                <property name="text">
                 <string>Assignment</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTabWidget" name="tab_run">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Batch</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QTreeView" name="tree_batch"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>Output</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QTextEdit" name="te_ouput"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_add_to_batch">
             <property name="text">
              <string>Add to Batch</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_remove">
             <property name="text">
              <string>Remove Task</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_run">
             <property name="text">
              <string>Run</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btn_load_batch">
         <property name="text">
          <string>Load Batch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_save_batch">
         <property name="text">
          <string>Save Batch</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layout_status_bar"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_help</tabstop>
  <tabstop>scenarios_tree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
